packages/components/Notification/src/Notification.helper.tsx

```diff
@@ -60,10 +60,6 @@ export const NotificationButton = stagedComponent((props: NotificationButtonProp
       backgroundColor: 'transparent',
       color: props.color,
       iconColor: props.color,
-      fontSize: 15,
-      fontWeight: '600',
-      fontLineHeight: 20,
-      fontLetterSpacing: -0.23, // iOS only prop
```

Comment:
Hmm shouldn't these props be getting replaced with something inside the 'subtle' object, rather than the 'medium" object?

Reply:
If we put these values in 'subtle', they get overridden further down the line. It's similar to what we saw in #2448.